src/emit/expression.rs

```diff
@@ -1252,7 +1321,12 @@ pub(super) fn expression<'a, T: TargetRuntime<'a> + ?Sized>(
 
             gv.as_pointer_value().into()
         }
-        Expression::ArrayLiteral(_, ty, dims, exprs) => {
+        Expression::ArrayLiteral {
+            ty,
+            lengths: dimensions,
```

Comment:
How about using `dimensions` instead of `lengths`?

Reply:
I agree. Fixed.